refact_webgui/webgui/tab_finetune.py

```diff
@@ -110,6 +126,23 @@ def __init__(self, model_assigner: ModelAssigner, *args, **kwargs):
         self.add_api_route("/tab-finetune-training-get", self._tab_finetune_training_get, methods=["GET"])
         self._model_assigner = model_assigner
 
+    async def _rename_run(self, post: RenameRunPost):
+        dir_loras = env.DIR_LORAS
+        runs = get_finetune_runs()
+        if post.run_id_new in [r["run_id"] for r in runs]:
+            raise HTTPException(status_code=400, detail=f"run with name {post.run_id_new} already exists")
+
+        run = [r for r in runs if r["run_id"] == post.run_id_old]
+        if not run:
+            raise HTTPException(status_code=400, detail=f"run {post.run_id_old} not found")
+
+        path_new = os.path.join(dir_loras, post.run_id_new)
+        os.rename(os.path.join(dir_loras, run[0]["run_id"]), path_new)
```

Comment:
rename can affect already running lora
we should send signal for worker to restart or so + change lora name in active loras